javascript/javascript2/week2/currencyCnvrtrWeek2.js

```diff
@@ -0,0 +1,132 @@
+// currencyConverter.js
+
+const exRates = {
```

Comment:
The brief says `creating an array of currency rate objects` so you need to create more objects like `exRates` and store them in an array. This will have an impact on your search function.

Reply:
Hi vdaguenet, You mean I need to put more currency rates in exRates. Now there are only 4 currency rates.

Reply:
No, you will need more currency objects in an array. For example:
```js
const allRates = [
  {
      "base": "EUR",
      "rates": {
          "USD": 1.23396,
          "DKK": 1.314
      }
  },
 {
      "base": "USD",
      "rates": {
          "CHN": 0.32,
          "ABC": 1.23
      }
  },
 {
      "base": "SEK",
      "rates": {
          "CDE": 0.32,
          "DEF": 1.23
      }
  },
  // etc
]
```